keeper/keeper.go

```diff
@@ -50,7 +50,9 @@ type Keeper struct {
 	WormchainChannel collections.Item[string]
 	GuardianSets     collections.Map[uint32, types.GuardianSet]
 	Sequences        collections.Map[[]byte, uint64]
-	VAAArchive       *collections.IndexedMap[[]byte, collections.Pair[string, bool], VAAArchiveIndexes]
+	// TODO: what is the purpose of the bool?
```

Comment:
This is a very good comment actually! Before, this was simply a VAA hash -> bool map, but then we updated it in #1 to allow querying by the VAA message ID as well. We can remove the boolean, but that should be done in a separate PR, as well as brainstorming how we migrate the existing state on testnet 😄